### PR TITLE
Help to choose between Let's Encrypt or Standard certificates

### PR DESCRIPTION
**Pull request by weppos**
I created an article to help the customers select which certificate is the best for them. The idea is to link the article in this page:

screen_shot_2016-11-15_at_12_04_54

I wonder what's the best way to render this section here. Is it a table appropriate, or would it be more appropriate to add each question as new heading, and a short explanation as a response?

/cc @dnsimple/dnsimple

### CODE REVIEW

**Comment by weppos** [comment]
Can you think of any other requirements/questions to add?

File: content/articles/standard-vs-letsencrypt.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Cost | Free | $20-$100
+Custom CSR | Not Supported | Supported
+
+## What is the right certificate for me? {#whichone}
+
+The following list of questions may help you to determine what is the best certificate for you. The answer is based on possible limitations of a specific certificate to fulfill the requirement. This also means that if you combine two or more questions (requirement), the results may conflict each other.
+
+| Requirement | Answer |
+|-------------+--------|
+You want to secure a domain name. | **Let's Encrypt** &middot; **Standard**
```
Comment (sbastn) [inline content/articles/standard-vs-letsencrypt.markdown]: I would replace "&middot;" with "or"

File: content/articles/letsencrypt.markdown [hunk 2]
```diff
@@ -32,9 +32,25 @@ Let's Encrypt is an innovative certificate authority from several point of views
...
+## Differences between Let's Encrypt and Standard SSL certificates
+
+The table below summarizes the most important differences between Let's Encrypt and Standard SSL certificates.
+
+|               | Let's Encrypt | Standard      |
+|---------------+---------------+---------------|
+| Certificate Expiration | 90 days | 1-3 years |
+| Single names | Supported | Supported |
+| Wildcard names | Not Supported | Supported |
+| Multi-domain (SAN) | Supported by default | Supported only by specific products |
```
Comment (sbastn) [inline content/articles/letsencrypt.markdown]: Should we mention plan support like you do on the `standard-vs-letsencrypt` page?

File: content/articles/standard-vs-letsencrypt.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Please note that the table only reflects the status of the current DNSimple offering. Some features may be available at the Certificate Authority, but currently not supported by DNSimple.
+</note>
+
+|               | Let's Encrypt | Standard      |
+|---------------+---------------+---------------|
+Certificate Expiration | 90 days | 1-3 years
+Single names | Supported | Supported
+Custom Single names | Supported (on certain plans) | Supported
+Wildcard names | Not Supported (by CA) | Supported
+Same-domain multi-names (SAN) | Supported (on certain plans) | Not Supported
```
Comment (sbastn) [inline content/articles/standard-vs-letsencrypt.markdown]: Do we want to be more specific about what support on each plan by mentioning the plan?

File: content/articles/letsencrypt.markdown (hunk 2 above)
Comment (weppos) [inline content/articles/letsencrypt.markdown]: Here it's about generic differences, not specific to DNSimple.

File: content/articles/standard-vs-letsencrypt.markdown (hunk 3 above)
Comment (weppos) [inline content/articles/standard-vs-letsencrypt.markdown]: We have a section called `#limitations` in the `letsencrypt.markdown` page. I'd avoid for now to duplicate the specific details (to help maintenance).

File: content/articles/standard-vs-letsencrypt.markdown (hunk 1 above)
Comment (weppos) [inline content/articles/standard-vs-letsencrypt.markdown]: ✔️

File: content/articles/standard-vs-letsencrypt.markdown (hunk 3 above)
Comment (sbastn) [inline content/articles/standard-vs-letsencrypt.markdown]: Sounds good then.

File: content/articles/letsencrypt.markdown (hunk 2 above)
Comment (sbastn) [inline content/articles/letsencrypt.markdown]: Sounds good.

**Pull request merged**